assets/src/edit-story/elements/video/edit.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { elementFillContent, getMediaProps } from '../shared';
+import { useStory } from '../../app';
+import StoryPropTypes from '../../types';
+import EditPanMovable from '../shared/editPanMovable';
+import { videoWithScale } from './util';
+import ScalePanel from './scalePanel';
+
+const Element = styled.div`
+	${ elementFillContent }
+`;
+
+const CropBox = styled.div`
```

Comment:
CropBox seems to be defined in the image element. Can we make generic and reusable.

Reply:
Done.